include/veriblock/pop/blockchain/base_block_tree.hpp

```diff
@@ -157,89 +157,7 @@ struct BaseBlockTree {
    * state.
    */
   virtual bool loadBlock(const stored_index_t& index, ValidationState& state) {
```

Comment:
Remove this function, just leave only loadBlockForward loadBlockBackward and make them `public` and `virtual` as with the loadBlock